examples/main/README.md

```diff
@@ -241,6 +241,22 @@ The `--mirostat-ent` option sets the Mirostat target entropy (tau), which repres
 
 Example usage: `--mirostat 2 --mirostat-lr 0.05 --mirostat-ent 3.0`
 
+### XTC Sampling
+
+-   `--xtc-p N`: Sets the chance for token removal (checked once on sampler start) (default: 0.5).
+-   `--xtc-t N`: Sets a minimum probability threshold for tokens to be removed (default: 0.1).
+-   `--xtc-t-max N`: Sets a maximum probability threshold for tokens to be removed (highly expetrimental) (default: 1.0).
```

Comment:
```suggestion
-   `--xtc-t-max N`: Sets a maximum probability threshold for tokens to be removed (highly experimental) (default: 1.0).
```

Reply:
In regards to sampling queue: XTC should be the last, right? I ask because it makes sense to have temperature both before and after XTC, but it can change the result significantly - especially with low temperature values (like recommended 0.3 for Nemo). I feel like XTC should go before temperature (bur after other truncating samplers, of course) because XTC is a truncating sampler too.